CaT/src/Filter/Predicates/PredicateIn.php

```diff
@@ -28,7 +28,7 @@ public function getList() {
 	}
 
 	public function fields() {
-		$fields = $this->addPossibleFieldsToFields($this->list, array());
-		return $this->addPossibleFieldsToFields($this->value, $fields);
+		$fields = $this->addPossibleFieldsToFields($this->list->values(), array());
+		return $this->addPossibleFieldsToFields(array($this->value), $fields);
```

Comment:
How is this test justified? Did it work before? If not, is there a test for this?